mlflow/utils/autologging_utils/safety.py

```diff
@@ -539,7 +541,16 @@ def call_original(*og_args, **og_kwargs):
                 else:
                     return original(*args, **kwargs)
 
-    _wrap_patch(destination, function_name, safe_patch_function)
+    new_patch = _wrap_patch(destination, function_name, safe_patch_function)
+    _store_patch(autologging_integration, new_patch)
+
+
+def revert_patches(autologging_integration):
```

Comment:
Can we add a small docstring for this method?

Reply:
Done